src/App.tsx

```diff
@@ -1,95 +1,121 @@
-import VariantsForDiseaseCardContainer from "./components/VariantsForDiseaseCardContainer";
 import DrugsForProteinCardContainer from "./components/DrugsForProteinCardContainer";
-import DiseasesForDrugsCardContainer from "./components/DiseasesForDrugsCardContainer";
+import LayoutTemplate from "./layout/LayoutTemplate";
+import DiseaseContainer from "./components/DiseaseContainer";
+import InteractionsForProteinCardContainer from "./components/InteractionsForProteinCardContainer";
+import logo from "./svg/uniprot-rgb.svg";
+
+import "./App.css";
+import ProteinContainer from "./components/ProteinContainer";
+import DrugsForDiseaseContainer from "./components/DrugsForDiseaseContainer";
+import { ContextObj, Context } from "./types/context";
 import ProteinsForDrugsCardContainer from "./components/ProteinsForDrugsCardContainer";
+import VariantsForDiseaseContainer from "./components/VariantsForDiseaseContainer";
 
 class App extends Component {
   render() {
     return (
       <FranklinSite>
         <BrowserRouter basename={process.env.PUBLIC_URL}>
-          <div className="wrapper">
-            <Header
-              className="main-header"
-              logo={<img src={logo} alt="logo" width={120} height={50} />}
-            />
-            <div className="main-content">
-              <Switch>
-                <Route
-                  path={"/"}
-                  exact={true}
-                  render={(props: any) => {
-                    return (
-                      <ProteinsForDiseaseCardContainer
-                        identifier="Alzheimer disease"
-                        {...props}
-                      />
-                    );
-                  }}
-                />
-                <Route
-                  path={`/${Context.DISEASE}/:id`}
-                  component={ProteinsForDiseaseCardContainer}
-                  exact={true}
-                />
-                <Route
-                  path={`/${Context.DISEASE}/:id/${Context.PROTEIN}`}
-                  component={DiseasesForProteinCardContainer}
-                />
-                <Route
-                  path={`/${Context.PROTEIN}/:id/${Context.DISEASE}`}
-                  component={ProteinsForDiseaseCardContainer}
-                />
-                <Route
-                  path={`/${Context.PATHWAY}/:id/${Context.PROTEIN}`}
-                  component={PathwaysForProteinCardContainer}
-                />
-                <Route
-                  path={`/${Context.VARIANT}/:id/${Context.PROTEIN}`}
-                  component={VariantsForProteinCardContainer}
-                />
-                <Route
-                  path={`/${Context.INTERACTION}/:id`}
-                  component={InteractionCardContainer}
-                />
-                <Route
-                  path={`/${Context.DRUG}/:id/${Context.DISEASE}`}
-                  component={DrugsForDiseaseCardContainer}
+          <Switch>
```

Comment:
Not sure why there is a switch here, looks like it might not be useful for one child route, especially since it's rendering a redirect.

The bunch of routes lower in this file might be cleaned up by using switches and nested routes, but I guess if it works like this it's fine. We just need to make sure that 2 routes might not match at the same time without it being intended (which is what switch is for, match the first route only)